lxc/auth.go

```diff
@@ -1270,12 +1270,10 @@ func (c *cmdPermissionList) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// If first argument has the format of a remote, use it as a remote.
```

Comment:
Why is the standard pattern of doing:

```go
	// Parse remote
	resources, err := c.global.ParseServers(args[0])
	if err != nil {
		return err
	}

	resource := resources[0]
	client := resource.server
```

Not sufficient here?

Reply:
Because if the remote is not specified (e.g. default) then the first argument will be one of the list filters e.g. `project=<project_name>`.

Reply:
I see `ParseServers` will be unable to parse the filter as a remote and return the default remote, but we still need to know which argument is a remote and which is a filter to parse the filters correctly.